templates/data/eventing-manager-data.yaml

```diff
@@ -2,20 +2,117 @@ templates:
+            volumes:
+              - name: config
+                configMapName: kaniko-build-config
+              - name: signify-secret
+                secretName: signify-dev-secret
+            volumeMounts:
+              - name: config
+                mountPath: /config
+                readOnly: true
+              - name: signify-secret
+                mountPath: /secret
+                readOnly: true
+          limits:
+            limits_memory: 3Gi
+            limits_cpu: 2
+          dind_job_k3d:
+            annotations:
+            labels:
+              preset-kyma-guard-bot-github-token: "true"
+              preset-kind-volume-mounts: "true"
+              preset-dind-enabled: "true"
+            image: europe-docker.pkg.dev/kyma-project/prod/testimages/e2e-dind-k3d:v20231004-3defcf03
+            command: "/init.sh"
+            request_memory: "4Gi"
+            request_cpu: "3"
+            limits_memory: "4Gi"
+            limits_cpu: "3"
+          job_default:
+            securityContext:
+              privileged: false
+              seccompProfile:
+                type: RuntimeDefault
         jobConfigs:
           - repoName: "github.com/kyma-project/eventing-manager"
             jobs:
+              - jobConfig:
+                  name: release-eventing-manager-build
+                  annotations:
+                    owner: skydivingtunas
+                    description: Job to build eventing module for a release.
+                  always_run: true
+                  labels:
+                    preset-signify-prod-secret: "true"
+                  args:
+                    - "--name=eventing-manager"
+                    - "--config=/config/kaniko-build-config.yaml"
+                    - "--context=."
+                    - "--dockerfile=Dockerfile"
+                    - '--tag={{`{{ .Env \"PULL_BASE_SHA\" }}`}}'
+                  branches:
+                    - ^\w+\d+\.\d+\.\d+(?:-.*)?$ #Watches for new Tag
+                inheritedConfigs:
+                  global:
+                    - jobConfig_default
+                    - jobConfig_postsubmit
+                  local:
+                    - job_build
+                    - limits
+              - jobConfig:
+                  name: release-eventing-manager-module-build
+                  annotations:
+                    owner: skydivingtunas
+                    description: Job to build eventing module for a release.
+                  always_run: true
+                  labels:
+                    preset-sa-kyma-push-images: "true"
+                    preset-bot-github-token: "true"
+                  env:
+                    KUSTOMIZE_VERSION: "v4.5.6"
+                    MODULE_REGISTRY: "europe-docker.pkg.dev/kyma-project/prod/unsigned"
+                    IMG: "europe-docker.pkg.dev/kyma-project/prod/eventing-manager:${PULL_BASE_SHA}"
+                  command: "./scripts/release.sh"
```

Comment:
I didn't find this script file [here](https://github.com/kyma-project/eventing-manager).

Reply:
it is [here](https://github.com/muralov/eventing-manager/tree/module-build-job). There is another PR in review containing this script. I added on-hold  to this PR until another one is merged.